test/controllers/propertyLinking/ClaimPropertyOccupancyControllerSpec.scala

```diff
@@ -117,13 +117,57 @@ class ClaimPropertyOccupancyControllerSpec extends VoaPropertyLinkingSpec with S
         val doc = Jsoup.parse(contentAsString(result))
         val error = doc.getElementsByAttributeValue("href", "#lastOccupiedDate-day")
         error.text() shouldBe
-          s"Date of ${if (isAgent) "your client's " else ""}last day as " +
+          s"Date of ${if (isAgent) "your client's " else "your "}last day as " +
             s"${if (cap == Owner) "owner" else if (cap == Occupier) "occupier" else "owner and occupier"} " +
             "of the property must be after " +
             s"${d._1.getDayOfMonth} ${d._1.getMonth.toString.toLowerCase.capitalize} ${d._1.getYear}"
       }
     }
   }
+  "The claim occupancy page" should "validate that the end date of ownership is not in the future" in {
```

Comment:
We need some sort of welsh tests for this as-well, especially with the ticket saying that something has been incorrectly translated into Welsh